### PR TITLE
fix: Blank Spaces In the Input Message

### LINKED ISSUE #348: Empty Message passed in the chat
### Description

The Input field is not checking whether the input message is empty. 

### Screenshots

Empty

### Additional information

_No response_

### 🥦 Browser

Google Chrome

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by ArnabBCA**
# fixes #348 

# 👨‍💻 Changes proposed(What did you do ?)
Trimmed the Input Message To Remove Extra Black Spaces

# ✔️ Check List (Check all the applicable boxes)

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.

##  Note to reviewers

# 📷 Screenshots
Before
Empty
After
Empty Fixed

### CODE REVIEW

File: client/src/components/Chat.jsx [hunk 1]
```diff
@@ -215,7 +215,7 @@ const Chat = () => {
 
         socket.emit(NEW_EVENT_TYPING, { chatId: app.currentChatId, isTyping: false });
         const d = new Date();
-        let message = inputRef.current.value;
+        let message = inputRef.current.value.trim();        // Trim the message to remove the extra spaces
```
Comment (Dun-sin) [inline client/src/components/Chat.jsx:218]: not a good way to fix it, check if the message is empty and then do a return

**Review by Dun-sin — CHANGES_REQUESTED** [review]

File: client/src/components/Chat.jsx (hunk 1 above)
Comment (ArnabBCA) [reply client/src/components/Chat.jsx:218]: That  is already checked in the if condition

Comment (ArnabBCA) [reply client/src/components/Chat.jsx:218]: Check Line 255

Comment (ArnabBCA) [reply client/src/components/Chat.jsx:218]: Hey Kindly Check line No 255 I think there is no need to check the condition again

Comment (Dun-sin) [reply client/src/components/Chat.jsx:218]: then your pr should be fixing why it's not working instead of this

Comment (Dun-sin) [reply client/src/components/Chat.jsx:218]: which means it should check if it's empty spaces, but i guess that fixes it

**Review by Dun-sin — APPROVED** [review]
You did great, thanks for contributing, I hope you will stick around and continue to contribute to this project.

Consider giving this project a star and joining the community discord server for more resources and opportunities to connect with others. [👉🏽here👈🏽](https://discord.com/invite/ufcysW9q23)

File: client/src/components/Chat.jsx (hunk 1 above)
Comment (ArnabBCA) [reply client/src/components/Chat.jsx:218]: > not a good way to fix it, check if the message is empty and then do a return

Hey Kindly Check line No 255 I think there is no need to check the condition again it is already present in the if condition

**Pull request merged**